backend/orders/serializers.py

```diff
@@ -1,14 +1,20 @@
 from rest_framework import serializers
-from orders.models import Order
+from .models import Order
 
 
 class OrderSerializer(serializers.Serializer):
     id = serializers.IntegerField(read_only=True)
-    customer = serializers.IntegerField()
-    description = serializers.CharField(max_length=100, default='')
-    status = serializers.BooleanField(default=False)
-    acceptedDelivery = serializers.BooleanField(default=False)
-    deliveryMan = serializers.IntegerField()
+    orderId = serializers.IntegerField()
```

Comment:
Same situation, why should we keep 2 different ids of order

Reply:
first id - db row number
second id - order id